go.mod

```diff
@@ -150,6 +149,8 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
 
+require github.com/google/go-github/v48 v48.2.0
```

Comment:
Let's please change the version directly inline in l.20

Reply:
dependabot does not update this version on the left side

Reply:
done, PTAL @sbueringer

Reply:
Ah good point.